src/helpers/issue.ts

```diff
@@ -338,6 +339,7 @@ export const getAssignedIssues = async (username: string) => {
 };
 
 export const getOpenedPullRequestWithNoReviewsOver24HoursPassedAfterCreated = async (username: string) => {
```

Comment:
To increase code clarity I recommend splitting this into separate functions `getOpenedPullRequests` 
- and then filter by no reviews i.e. `noReviews` 
- and then filter again by `timePassed`